tests/bouncer/util_test.py

```diff
@@ -7,7 +7,13 @@ def test_parse_document_raises_if_no_uri():
     with pytest.raises(util.InvalidAnnotationError) as exc:
```

Comment:
There are a bunch more code style mistakes in this file (still no idea why Hound doesn't pick it up, maybe it isn't configured for this repo). Just briefly looking the output for `pep8 tests/bouncer/util_test.py` it looks like most of the errors are because of changes in this branch. Those should be cleaned up.